src/main/java/hudson/plugins/sidebar_link/LinkAction.java

```diff
@@ -65,7 +65,7 @@ public LinkAction(String urlName, String displayName, String iconFileName) throw
         }
 
         if(StringUtils.isBlank(iconFileName)) {
-            iconFileName = "static/efbf17e4/images/16x16/help.png";
+            iconFileName = "icon-help icon-md";
```

Comment:
This would cause LinkAction.getIconFileName() to return "icon-help icon-md". Such a space-delimited list doesn't seem valid according to the [documentation of Action.getIconFileName() in Jenkins 2.332.2](https://github.com/jenkinsci/jenkins/blob/jenkins-2.332.2/core/src/main/java/hudson/model/Action.java#L80-L99), which you set as the minimum version.

Reply:
That'll work perfectly fine on the new baseline; former references the .png, latter the icon class:
![](https://i.imgur.com/DEMLIMw.png)

Reply:
Is it documented or does it work by accident?

Reply:
@NotMyFault the _current_ you are showing is also the SVG is started this PR with.
@KalleOlaviNiemitalo here https://www.jenkins.io/doc/developer/views/icon-path-to-icon-class-migration/

Reply:
> @NotMyFault the current you are showing is also the SVG is started this PR with.

Yes, "current" mirrors your proposal, "new" is my proposed change by using the icon class..